### PR TITLE
IObit.RandomPasswordGenerator version 1.3

### PR DESCRIPTION
**Pull request by KarbitsCode**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/159895)

### CODE REVIEW

**Comment by stephengillie** [comment]
Verify:
image

Hi @KarbitsCode,

The application displays `1.3` while the installer writes `1.0` to the Registry. 
- A `DisplayVersion: "1.0"` should allow the package manager to match the Registry data, while displaying the same version `1.3` as the application. 
  - If the `DisplayVersion` is always `1.0`, then we could only have 1 copy in the repository, or the package manager could confuse them, causing issues. 
- Or the developer might update one or both of these to match, as another option.

What are your thoughts?

**Comment by KarbitsCode** [comment]
Maybe we can turn down the version in the manifest to match with the registry?

**Comment by KarbitsCode** [comment]
Or maybe we just cancel/close this PR, because this software is abandonware and I don't think the developer will update the installer anytime soon

**Comment by KarbitsCode** [comment]
> * If the `DisplayVersion` is always `1.0`, then we could only have 1 copy in the repository, or the package manager could confuse them, causing issues.

This might be a good idea, because this software is abandonware and I don't think the developer will update the installer anytime soon

File: manifests/i/IObit/RandomPasswordGenerator/1.3/IObit.RandomPasswordGenerator.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: IObit.RandomPasswordGenerator
+PackageVersion: "1.3"
+InstallerType: inno
+Installers:
+- Architecture: x86
+  InstallerUrl: https://cdn.iobit.com/dl/rpg-setup.exe
+  InstallerSha256: 8322FB43BFFEE31E64E8AE7F9FA39D60999B3AF70A9E85CADBF5E6F2111F20C4
+ManifestType: installer
```
Comment (stephengillie) [inline manifests/i/IObit/RandomPasswordGenerator/1.3/IObit.RandomPasswordGenerator.installer.yaml:11]:
```suggestion
AppsAndFeaturesEntries:
- DisplayVersion: "1.0"
ManifestType: installer
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
I'm hoping that the syntax is correct. I haven't made this specifc change before, so there's a chance it might error.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**